how-to-guides/bubs-testnet.md

```diff
@@ -67,27 +67,12 @@ Remember, Bubs Testnet is a testing environment!
 Bridging is a process that enables the transfer of assets between
 different blockchains.
 
-To bridge between Ethereum Sepolia and Bubs Testnet,
-visit the [Bubs Testnet bridge](https://bubs-sepolia.bridge.caldera.xyz/).
+To bridge between Ethereum Sepolia and Bubs Testnet, visit the [Bubs Testnet bridge](https://bubs-sepolia.bridge.caldera.xyz/).
 
 ### Faucet
 
-To visit the Bubs testnet faucet, go to
-[`https://bubs-sepolia.hub.caldera.xyz/`](https://bubs-sepolia.hub.caldera.xyz/)
-and click the "Faucet" tab.
-
-### Explorer
-
-To visit the explorer, go to
-[`https://bubs-sepolia.explorer.caldera.xyz/`](https://bubs-sepolia.explorer.caldera.xyz/).
-
-### Status
-
-To see the status and uptime information for Bubs,
-[visit the status page](https://bubs-sepolia.betteruptime.com/).
+To visit the Bubs testnet faucet, go to <https://bubstestnet.com/> and click the "Request Faucet Funds on Bulbs rollup" button.
```

Comment:
unclear if these exist now

Reply:
let's leave the bubstestnet page as an historical archive, this doesn't exist anymore

Reply:
2ef5023dea665267b37e6e83ef591133487e8988